textBasedAdventure.js

```diff
@@ -1,5 +1,156 @@
-const readline = require('readline-sync')
+const readline = require('readline-sync');
+let titleCard = readline.question("CAVE HERO! PRESS ENTER TO BEGIN!");
+console.log(`${titleCard}`);
+let nameInput = readline.question("Enter your name: ");
+console.log(`Greetings ${nameInput}!`);
 
-let nameInput = readline.question("Enter your name: ")
+///STATS
+let health = 100
+let damagePoints = 20
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+let goblinHealth = 50
+let GoblinKingHealth = 100
+let goblinMallet = 25
+
+let deathBatHealth = 30
+let unholySpiritHealth = 50
+let finalDragonHealth = 200
+
+let swordDamage = damagePoints + 20
+let sword = swordDamage
+let caveHeroWeapons = []
+let caveHero =  {
+    name: `${nameInput}`,
+    health: health,
+    atk: damagePoints,
+    weapon: caveHeroWeapons,  
+}
+
+let checkStatus = caveHero
+
+
+let overviewStatement = "We need your help. Our princess has been captured, and we need you to rescue her. Can we count on you?"
+let missionStart = readline.question(`${overviewStatement}`)
+
+missionStart = missionStart.toLowerCase()
+let caveEntrance = readline.question("You arrive at the entrance of a dimly lit cave aligned with torches. From the torchlight you see there are three tunnel entrances: One to the left, one to the right, and one to the center. Which tunnel will you enter? Enter LEFT, CENTER, RIGHT OR CHECK STATUS: ")
+caveEntrance = caveEntrance.toLowerCase()
+
+let directionChoice2 = "back"
+
+const goBack = (str) => {
+    // str = str.toLowerCase()
+    let back = readline.question("Type back to go back")
+    if (back === directionChoice2) {
+        return caveEntrance
+    }
+    return back
+}
+
+function startGame(choice) {
```

Comment:
For the future, I recommend using ES6 syntax to write functions.

const startGame = (choice) => {
    // your code here
}